alerting/src/test/kotlin/org/opensearch/alerting/resthandler/MonitorRestApiIT.kt

```diff
@@ -888,4 +894,47 @@ class MonitorRestApiIT : AlertingRestTestCase() {
         val trigger = randomQueryLevelTrigger(actions = listOf(action))
         return randomQueryLevelMonitor(triggers = listOf(trigger))
     }
+
+    @Throws(Exception::class)
+    fun `test search monitors only`() {
+
+        // 1. create monitor
+        val monitor = createRandomMonitor()
+        val createResponse = client().makeRequest("POST", ALERTING_BASE_URI, emptyMap(), monitor.toHttpEntity())
+        assertEquals("Create monitor failed", RestStatus.CREATED, createResponse.restStatus())
+
+        // 2. create destination
+        val chime = Chime("http://abc.com")
+        val destination = Destination(
+            type = DestinationType.CHIME,
+            name = "test",
+            user = randomUser(),
+            lastUpdateTime = Instant.now(),
+            chime = chime,
+            slack = null,
+            customWebhook = null,
+            email = null
+        )
+        val response = client().makeRequest(
+            "POST",
+            LEGACY_OPENDISTRO_DESTINATION_BASE_URI,
```

Comment:
Should we start using the `OPENSEARCH` URI?

Reply:
I agree, we should start using the OpenSearch URI.

Reply:
Changed. thanks